docs/UserGuide.md

```diff
@@ -225,3 +240,9 @@ Action | Format, Examples
 **Find** | `find_session KEYWORD`<br><br>e.g. `find_session John`
 **Add** | `add_session n/STUDENT_NAME d/DATE t/TIME k/DURATION s/SUBJECT f/FEE`<br><br> e.g. `add_session n/John Doe d/2021-01-01 t/18:00 k/120 s/Biology f/80`
 **Delete** | `delete_session n/STUDENT_NAME i/SESSION_INDEX`<br><br>e.g. `delete_session n/John Lee i/1`
+
+**Others**
```

Comment:
I think the header can be "Fees"?

Reply:
Haha yeah I tot of before, but then I think at most there will only be 2 commands, one to get monthly fee for a particular month, one to get past 3 months fee. So not sure if it should be a section by itself. Although I think doesn't hurt to change ba. I will make the changes first then.